WordPress/Classes/Services/AccountSettingsService.swift

```diff
@@ -53,8 +48,22 @@ struct AccountSettingsService {
         }
     }
 
+    /// Emits a value when the settings for the associated account change.
+    var settings: Observable<AccountSettings?> {
+        let notificationCenter = NSNotificationCenter.defaultCenter()
+        let notificationObserver = notificationCenter.rx_notification(NSManagedObjectContextDidSaveNotification, object: context)
+        // This was the simplest implementation. If performance is an issue, we could try
+        // adding `distinctUntilChanged` or `filter` on the notification userInfo and only
+        // emit if the changed objects include the observed account.
+        return notificationObserver.map(getSettings).startWith(getSettings())
```

Comment:
So if I'm understanding this correctly then every time a context is saved `getSettings` is called? I assume the rx_notification observer remove itself after the view controller is deallocated, right?